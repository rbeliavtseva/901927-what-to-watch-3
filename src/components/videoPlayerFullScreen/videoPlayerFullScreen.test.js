import React from "react";
import renderer from "react-test-renderer";
import VideoPlayerFullScreen from "./videoPlayerFullScreen.jsx";

const movie = {
  title: `Fight Club`,
  poster: `http://picsum.photos/id/1020/300/150`,
  id: 19,
  preview: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  genre: `Drama`,
  year: 1999,
  fullMovie: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

it(`Render VideoPlayerFullScreen`, () => {
  const tree = renderer
    .create(<VideoPlayerFullScreen
      onFullScreenClose={() => {}}
      movie={movie}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

