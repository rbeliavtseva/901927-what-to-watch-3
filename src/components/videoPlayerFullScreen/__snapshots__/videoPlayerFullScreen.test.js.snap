// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayerFullScreen 1`] = `
<div
  className="player"
  style={
    Object {
      "zIndex": 9999,
    }
  }
>
  <video
    className="player__video"
    height="100%"
    muted={false}
    onLoadedMetadata={null}
    onTimeUpdate={null}
    poster="http://picsum.photos/id/1020/300/150"
    width="auto"
  >
    <source
      src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
      type="video/mp4"
    />
    <source
      src="https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4"
      type="video/webm"
    />
  </video>
</div>
`;
